File: SpriteKitLib/src/main/java/br/com/insanitech/spritekit/opengl/context/GL10ContextFactory.java
package br.com.insanitech.spritekit.opengl.context;

import javax.microedition.khronos.egl.*;

/**
 * Created by anderson on 6/30/15.
 */
public class GL10ContextFactory extends GLContextFactory {
    @Override
    public EGLContext createContext(EGL10 egl, EGLDisplay display, EGLConfig eglConfig) {
        glVersion = GLVersion.GL10;
        return super.createContext(egl, display, eglConfig);
    }
}

File: SpriteKitLib/src/main/java/br/com/insanitech/spritekit/opengl/context/GL10ContextFactory.kt
package br.com.insanitech.spritekit.opengl.context

import javax.microedition.khronos.egl.*

/**
 * Created by anderson on 6/30/15.
 */
internal open class GL10ContextFactory : GLContextFactory() {
    override fun createContext(egl: EGL10, display: EGLDisplay, eglConfig: EGLConfig): EGLContext? {
        this.glVersion = GLVersion.GL10
        return super.createContext(egl, display, eglConfig)
    }
}
